assessments/projects/rook/self-assessment.md

```diff
@@ -0,0 +1,374 @@
+# Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
+security of the project, determining gaps in their security, and preparing any security
+documentation for their users. This document is ideal for projects currently in the
+CNCF **sandbox** as well as projects that are looking to receive a joint assessment and
+currently in CNCF **incubation**.
+
+For a detailed guide with step-by-step discussion and examples, check out the free 
+Express Learning course provided by Linux Foundation Training & Certification: 
+[Security Assessments for Open Source Projects](https://training.linuxfoundation.org/express-learning/security-self-assessments-for-open-source-projects-lfel1005/).
+
+# Self-assessment outline
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+A table at the top for quick reference information, later used for indexing.
```

Comment:
Please remove any generic content from the template

Reply:
Same commit as above: [cfbb19f](https://github.com/dhauss/tag-security/commit/cfbb19f0ab420fbffd3366d72b2d3c037085bf8d)

Reply:
Still showing up on latest for me. Likely got restored by a later commit.

Reply:
Honestly think I just missed it- removed it for real now: [this commit](https://github.com/dhauss/tag-security/commit/ab38ebc933b142aa6f432e7b7516f59a58975920)